### ISSUE TITLE
Additional candidate metadata

### ISSUE THREAD

**Issue by aalmiray** [sdkman/sdkman-java-sdk#6]
I know `gvm-sdk` simply consume data comign from `gvm`'s server side however I thought this would be a good place to start this discussion. Say someone builds a `griffon` client for `gvm-sdk` (hint, hint :wink:); in order to make the app more appealing one would think it may display a logo and some other additional data that the `gvm` REST API does not expose (yet). Specifically I'm thinking of (at least) the following items:
- name. We've got this already, that's the candidate name.
- logo. Qualified list of URLs with small, medium, large, huge as qualifiers for example.
- summary. One liner stating the candidate's purpose.
- description. A few paragraphs (up to 3 maybe) in  render friendly format (markdown perhaps?) with further details on the candidate.
- url.

WDYT?

**Comment by marc0der** [comment]
Love it! I'll add this to the backlog for my new gvm server (written in shiny new spring-boot).

**Comment by pledbrook** [comment]
So how many implementations has the gvm server now seen? :smile: BTW, what are the advantages of Spring Boot in this case? Not knocking the decision, just interested in the reasoning.

**Comment by aalmiray** [comment]
I bet @smaldini would love to see an implementation using https://github.com/reactor/reactor :wink:

:squirrel:

**Comment by pledbrook** [comment]
Nah, it has to be Reactor _with_ something. I think he favours Ratpack atm.

**Comment by marc0der** [comment]
@pledbrook this is going to be the first rewrite of the server. I favour spring-boot because it's well suited for micro services, allows me to write in groovy, and builds a fat jar making deployment on paas a breeze.

**Comment by pledbrook** [comment]
Ah, I thought you'd tried Ratpack and DropWizard (which also fit those benefits). I wondered if it was the Spring support itself that was significant.

**Comment by marc0der** [comment]
Yes, that's also a big seller. Spring Data for Mongo is very very useful.

DropWizard was the other great contender. Ratpack not so much because it seems closer to web framework than MSA. Also, when I last used it, it didn't generate a fat jar, but an awkward gradle script stuck deep in the build folder.

**Comment by aalmiray** [comment]
Right. Ratpack cna be very minimalistic. Good thing a Ratpack project is a valid Gradle project so you can use https://aalmiray.github.io/gradle-plugins/plugins/gradle-fatjar-plugin.html :wink:

**Comment by marc0der** [comment]
Thats true, but I still favour spring-boot more :D

On 11 March 2014 16:12, Andres Almiray notifications@github.com wrote:

> Right. Ratpack cna be very minimalistic. Good thing a Ratpack project is a
> valid Gradle project so you can use
> https://aalmiray.github.io/gradle-plugins/plugins/gradle-fatjar-plugin.html [image:
> :wink:]
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/gvmtool/gvm-sdk/issues/6#issuecomment-37314400
> .

## 

Marco Vermeulen
+44 7757 510 608
